core-bundle/assets/controllers/ace-controller.js

```diff
@@ -43,6 +43,11 @@ export default class extends Controller {
         this.container.remove();
     }
 
+    colorChange(event) {
+        console.log(event);
```

Comment:
```suggestion
```

Reply:
@m-vo that was before figuring out that the color scheme controller was missing something. Good catch! 🙈